app/modules/main/containers/Sections/Account/Overview/Ram.js

```diff
@@ -50,7 +52,11 @@ class AccountOverviewRam extends Component<Props> {
                       marginTop: '0.35em',
                     }}
                   >
-                    A <strong>size-based</strong> resource an account can use to <strong>store data</strong> within smart contracts.
+                    <div
+                      dangerouslySetInnerHTML={{
```

Comment:
Another instance of `dangerouslySetInnerHTML` - was this required, and if so, can we do it another way?